app/colculater.py

```diff
@@ -0,0 +1,60 @@
+import math
+import datetime
+
+from app.customers import Customers
+
+
+def calculater(customer: Customers,
+               shops: list,
+               fuel_price: float) -> None:
+    print(f"{customer.name} has {customer.money} dollars")
+
+    customer_location = customer.location
+    fuel_consumption = customer.car["fuel_consumption"]
+
+    total_price = 1_000_000
```

Comment:
"crutch"

Reply:
you should fix it

Reply:
'crutch' Do you want say me about remove the pass?